content/curating/kata.md

```diff
@@ -0,0 +1,92 @@
+
+### Satisfaction vote
+
+Users with [sufficient privileges][privileges-reference] are able to vote on kata quality. They can cast one of three possible votes:
+
+- **Satisfied**
+- **Somewhat satisfied**
+- **Not satisfied**
+
+If you think the kata itself is valuable but you simply have some issues with it, then you should select "Somewhat" and create an issue instead. If you thought the kata is really great, other than the issues, then you can choose "Very". Please do not choose "None" simply because the kata has an issue but it is otherwise a good kata. The satisfaction rating is meant to indicate quality of kata as a concept, not its current state of condition in terms of open issues which can be fixed.
+
+### Difficulty rating vote
+
+Kata author assigns initial difficulty rank estimation to their kata, but they do not decide on its final rank. Difficulty rank is estimated from votes cast by users who solved the kata while it's in beta phase. This means that every user who solved a beta kata can give their own opinion on its difficulty, helping in determining its final rank.
+
+### Review
+
+Before a beta kata is accepted and becomes available for users to train on, it needs to be reviewed and determined whether it holds up to Codewars quality standards.
+When reviewing a beta kata, users can perform the following steps:
+
+- **Reading the code and description**.
+- **Running the tests** against a couple of solutions, to verify if test suite gracefully handles some unexpected answers and eventual crashes of the solution, or if assertion messages are clear enough, etc.
+- **Reading the comments** to check if there are any remarks from other reviewers and whether they still hold or were addressed appropriately.
+- **Verifying** if the kata holds to [kata authoring guidelines][kata-authoring-guidelines].
+
+After a review is completed, the reviewer can finalize it with one of the following actions:
+
+- **Leaving a properly labeled comment** with remarks, suggestions and issues which should be addressed by the author.
+- **Fixing problems found during the review,** if privileges allow.
+- **Approving the kata**, if it fulfills all required conditions (see below).
+
+Users performing a review should hold to [Kata Curating Guidelines][kata-curating-guidelines].
+
+### Review comments
+
+When a reviewer writes a post pointing out the spotted issues which need to be fixed, it should be done in a way which actually instructs the author how to fix them. Sometimes, authors can be not aware that some specific issue might occur, or why this might be a problem, and reviewers with their short remarks leave them even more clueless. When a reviewer wants to signal such issue, it should be done in a way which would minimize the need for follow-up questions and additional explanations. Initially it might seem as more effort for reviewer, but this additional effort might be finally worth it if it helps to resolve the problems easier.
```

Comment:
```suggestion
When a reviewer writes a post pointing out the spotted issues which need to be fixed, it should be done in a way which actually instructs the author how to fix them. Sometimes, authors can be unaware that some specific issue might occur, or why it might be a problem, and reviewers with their short remarks leave them even more clueless. When a reviewer wants to signal such an issue, it should be done in a way which minimizes the need for follow-up questions and additional explanations. Initially it might seem as more effort from the reviewer, but this additional effort may ease the resolution of problems further down the line.
```